src/sentry/plugins/sentry_mail/models.py

```diff
@@ -123,6 +126,43 @@ def get_send_to(self, project):
             logger.debug('Tried to send notification to invalid project: %r', project)
             return []
 
+        if event:
+            owners, _ = ProjectOwnership.get_owners(project.id, event.data)
+            if owners != ProjectOwnership.Everyone:
+                if not owners:
+                    metrics.incr(
```

Comment:
@JTCunning feedback on the metric paths here?

Reply:
Made sine changes @JTCunning. How do you feel about it?

Reply:
This is logically correct, let's just reduce our code footprint by instantiating an `outcome='everyone' # everyone by default` variable above this `if` block, the metrics call we're currently commenting on and the one below it just do `outcome='empty'/'match'` and move the final metric call outside of the `else` block.